src/components/_common/illustration/IllustrationStatic.tsx

```diff
@@ -44,7 +42,7 @@ export const IllustrationStatic = ({
                 }}
             />
             <div
-                className={classNames(bem('icon'), bem('icon', 'icon2'))}
+                className={classNames(style.icon, 'icon2')}
```

Comment:
Finner ikke noe styling for icon2. Fjernes?

Reply:
Fjernet